crates/dojo-core/src/world/world_contract.cairo

```diff
@@ -6,12 +6,20 @@ use dojo::model::{ModelIndex, ResourceMetadata};
 use dojo::model::{Layout};
 use dojo::utils::bytearray_hash;
 
+#[derive(Drop, starknet::Store, Serde, Default, Debug)]
+pub enum Resource {
+    Model: (ClassHash, ContractAddress),
+    Contract: (ClassHash, ContractAddress),
+    Namespace,
+    World,
+    #[default]
+    None,
```

Comment:
Wondering if `None` is ok, or if we should change for `Unknown`/`Unregistered`. Very open question, no strict preference on my end. Was wondering what you guys think.

Reply:
Yes, why not ! Why one do you prefer ? Maybe `Unregistered` is the most explicit/specific term ?

Reply:
Yup let's go with `Unregistered`. 👍